app/routes.py

```diff
@@ -25,4 +37,29 @@ def post_kml():
     quoted_str = request.get_data().decode('utf-8')
     data = unquote_plus(quoted_str)
     kml_string = validate_kml_string(data)
-    return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
+    kml_admin_id = base64.urlsafe_b64encode(uuid.uuid4().bytes).decode('utf8').replace('=', '')
+    kml_id = base64.urlsafe_b64encode(uuid.uuid4().bytes).decode('utf8').replace('=', '')
+    timestamp = time.strftime('%Y-%m-%d %X', time.localtime())
+    executor = S3FileHandling(AWS_S3_REGION_NAME, AWS_S3_ENDPOINT_URL)
+    executor.upload_object_to_bucket(kml_id, kml_string, bucket_name=AWS_S3_BUCKET_NAME)
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    enforcer.save_item(kml_admin_id, kml_id, timestamp)
+    return make_response(
+        jsonify(
+            {
+                'code': 201,
+                'id': kml_admin_id,
+                'links':
+                    {
+                        'self': f'{SERVICE_URL}/kml/{kml_admin_id}',
+                        'kml': f'public.geo.admin.ch/{kml_id}'
+                    }
+            }
+        ),
```

Comment:
Based on our guideline https://github.com/geoadmin/doc-guidelines/blob/master/API.md#reserved-top-level-params-in-payloads it should be `"success": True` instead of `"code": 201`, @boecklic ?